packages/Webkul/Shipping/src/Shipping.php

```diff
@@ -27,17 +27,19 @@ public function collectRates()
 
         $this->removeAllShippingRates();
 
+        $tmp = [];
```

Comment:
Give some meaningful names here and a new line.

Reply:
OK